### PR TITLE
Added Rule Regarding Communication for Higher Standard Roles

### PR DESCRIPTION
**Pull request by FungiFellow**
## About the PR

Rules changed, see files

## Why / Balance

https://discord.com/channels/1168210010233376858/1304836825076072448
https://discord.com/channels/1168210010233376858/1305094703527497828
https://discord.com/channels/1168210010233376858/1305226421186068530

- #4872 should be merged with this

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

:cl: FungiFellow, Whisper
- add: Added Rule Requiring clear communication for Command/MP/SLs.

### CODE REVIEW

**Comment by mmails** [comment]
I personally second this

**Comment by snebl** [comment]
I personally don't see the need for this. If you don't understand something command said you could always ask for clarification. The rule seems overly restrictive for something so specific and flat-out prevents certain characters from certain roles for what could be an in-character genetic trait.

Does this really happen often enough and badly enough to where it would be necessary to add a new admin enforced rule about it, or is it that some people just don't like being talked down to by someone with a speech impediment?

**Comment by mmails** [comment]
> I personally don't see the need for this. If you don't understand something command said you could always ask for clarification. The rule seems overly restrictive for something so specific and flat-out prevents certain characters from certain roles for what could be an in-character genetic trait.
> 
> Does this really happen often enough and badly enough to where it would be necessary to add a new admin enforced rule about it, or is it that some people just don't like being talked down to by someone with a speech impediment?

The goal of this rule isn’t to restrict creativity or prevent certain characters from having unique quirks. We all enjoy the fun that personality and flavor bring to our characters. The intention here is simply to ensure that command roles can communicate information efficiently and clearly, especially during fast-paced or critical moments in the game where clarity is key for everyone involved.

Having a rule that encourages clearer speech for command characters helps make sure that instructions are understood the first time, without needing frequent clarifications, which can slow things down. We’ve had some feedback from players who find it challenging to follow orders when a command character uses overly whimsical language. Since command roles hold responsibilities that directly impact other players’ experiences, a more standardized communication style could make the roleplay smoother for all.

Of course, this rule doesn’t mean that all personality has to be stripped away. Characters can still have quirks and traits that make them unique—just maybe in ways that don’t compromise clarity when it matters most. This change is simply about balancing creativity with functionality to keep the experience engaging for everyone.

**Review by QuietlyWhisper — APPROVED** [review]

File: Resources/ServerInfo/Guidebook/_RMC14/Rules/Overview.xml [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
-  - (End-of-round grief, Zero tolerance) Suicides done at round-end without a valid roleplay reason are now prohibited and are counted as regular end-of-round grief, and will result in an immediate unappealable 3-hour ban.
-  - (General Roleplay Standards) Suicides can be done with a valid roleplay reason. Read the section for more details. An example is suiciding as a survivor in a last stand scenario, when it is clear that you will be captured by xenonids and there is no hope of fighting back.
+  - (Command/MP Roleplay Standards) You are expected to be able to communicate clearly when giving orders or acting on things related to marine law, like arrest handling and appeals.
+  - (General Roleplay Standards, metagaming) Surrounding a comms tower entirely with reinforced walls to prevent Xenos attacking it without T3s is not allowed.
+  - (General Roleplay Standards, Suicides) Suicides can be done with a valid roleplay reason. Read the section for more details. An example is suiciding as a survivor in a last stand scenario, when it is clear that you will be captured by xenonids and there is no hope of fighting back. Do not attempt then cancel a suicide, it is treated the same.
```
Comment (SolidusSnek) [inline Resources/ServerInfo/Guidebook/_RMC14/Rules/Overview.xml:10]: "Do not attempt then cancel a suicide, it is treated the same."
Change "then" to 'to'.

**Pull request merged**